### PR TITLE
Enabled 'copy image' context menu option in gallery and chat view and…

### PR DESCRIPTION
**Pull request by flipsimon**
… removed 'copy text' option if text is empty.

### CODE REVIEW

**Comment by flipsimon** [comment]
Yet to fix (but does not seem urgent):
If a message contains an image and (only) a link/email, the context menu options differ if one right-clicks on the image or on the text part of the message. When clicked on the image, the context menu offers options 'Copy image' and 'Copy text', when clicked on the text, the context menu offers 'Copy image' and 'Copy link/email'.

**Comment by flipsimon** [comment]
@Simon-Laux branch based on new_core_1.71. Hope that was correct?

File: src/main/deltachat/login.ts [hunk 1]
```diff
@@ -221,7 +221,6 @@ export function txCoreStrings() {
   strings[C.DC_STR_AUDIO] = tx('audio')
   strings[C.DC_STR_FILE] = tx('file')
   strings[C.DC_STR_ENCRYPTEDMSG] = tx('encrypted_message')
-  strings[C.DC_STR_STATUSLINE] = tx('pref_default_status_text')
```
Comment (Simon-Laux) [inline src/main/deltachat/login.ts:224]: technically this doesn't belong in the pr as it is for making the code work with the newest core,
which is already done in #2530

**Review by Simon-Laux — APPROVED** [review]

File: src/main/deltachat/login.ts (hunk 1 above)
Comment (flipsimon) [reply src/main/deltachat/login.ts:224]: Thanks for reviewing! So now I should remove the changes from this file from the PR even if the tests fail then?

Comment (Jikstra) [reply src/main/deltachat/login.ts:224]: The tests should only fail locally for you if you use a newer deltachat-node :)
